NEWS.md

```diff
@@ -2,12 +2,16 @@
 
 - Default TensorFlow version installed by `install_keras()` is now 2.11. 
 
-- All optimizers have been updated for keras/tensorflow version 2.11. 
-  Arguments to all the optimizers have changed. To access the previous optimizer implementations, 
-  use the constructors available at `keras$optimizers$legacy`.
-  For example, `keras$optimizers$legacy$Adam()` for the previous implementation of `optimizer_adam()`.
+- All optimizers have been updated for keras/tensorflow version 2.11.
+  Arguments to all the optimizers have changed. To access the previous
+  optimizer implementations, use the constructors available at
```

Comment:
So the new version won’t be backwards compatible and users have to use the reticulate way of accessing the legacy methods?

Reply:
I think the vast majority of users just call `optimizer_adam()` with zero arguments, or 1 argument for `learning_rate`. For those simple cases, the usage stays the same.

It is technically a small breaking change if users were using some of the long tail of optimizer arguments, but 

a) The breaking change is from upstream. We have to adapt either by freezing the R wrappers on the legacy implementation or propagating the breaking change through to the R API. I think it makes sense to match the R interface to the Python interface. 
b) The fix for backwards compatability is very simple, users can substitute `optimizer_adam(...)` with `keras$optimizers$legacy$Adam(...)`

Reply:
That makes sense. Sounds good!